containers/src/Kokkos_Bitset.hpp

```diff
@@ -108,6 +108,7 @@ class Bitset
   }
 
   /// assignment
+  KOKKOS_FORCEINLINE_FUNCTION
```

Comment:
Is there a particular reason we need `FORCEINLINE` instead of just `INLINE`?  If not, we could just do this:
```c++
KOKKOS_INLINE_FUNCTION
Bitset (const Bitset<Device>&) = default;
KOKKOS_INLINE_FUNCTION
Bitset& operator= (const Bitset<Device>&) = default;

// Respect the Rule of Five.

KOKKOS_INLINE_FUNCTION
Bitset (Bitset<Device>&&) = default;
KOKKOS_INLINE_FUNCTION
Bitset& operator= (Bitset<Device>&&) = default;
KOKKOS_INLINE_FUNCTION
~Bitset () = default;
```